src/components/trade/PriceSuggestions/PriceSuggestionItem.tsx

```diff
@@ -21,54 +21,67 @@ export interface Props {
 
 interface FormattedPrices {
   priceFormatted: string
-  invertedPriceFormatted: string
+  inversePriceFormatted: string
+  inversePriceRaw: string
+  priceRaw: string
```

Comment:
good call

Reply:
good call

I would expect raw being a BigNumber (judging by the name)
Since all these are "formatted" (all are strings), I would suggest:
- priceLabel
- inversePriceLabel
- priceValue
- inversePriceValue

I think, then the `<button>` is more clear